book/security.rst

```diff
@@ -1352,6 +1352,13 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
+
+.. code-block:: bash
+
+    $ composer require 'sensio/distribution-bundle:~3'
```

Comment:
We can omit the version number (it would pick the latest version). I also think we should add this in a new note box, like:

```rst
.. tip::

    The ``security:check`` command terminates with a non-zero exit code if
    any of your dependencies is affected by a known security vulnerability.
    Therefore, you can easily integrate it in your build process.

.. note::

    To enable the ``security:check`` command, make sure the
    `SensioDistributionBundle`_ is installed:

    .. code-block:: bash

        $ composer require sensio/distribution-bundle
```

Can you please update if you agree?